index.js

```diff
@@ -85,10 +85,51 @@ module.exports = class BemEntityName {
 
         return this._type;
     }
-
-    toString() { return this.id;  }
-    valueOf()  { return this._obj; }
-
+    /**
+     * Returns string representing the entity name.
+     *
+     * @returns {String}
+     * @example
+     * const BemEntityName = require('bem-entity-name');
+     * const name = new BemEntityName({ block: 'button' });
+     *
+     * console.log(`name: ${name}`); // button
+     */
+    toString() { return this.id; }
+    /**
+     * Returns object representing the entity name. Need to debug in some browsers.
+     *
+     * In some browsers `console.log()` calls `util.inspect()` on each argument.
+     * This method will be called to get custom string representation of the object.
+     *
+     * The representation object contains only `block`, `elem` and `mod` fields
+     * without private and deprecated fields (`modName` and `modVal`).
+     *
+     * @returns {Object}
+     * @example
+     * const BemEntityName = require('bem-entity-name');
+     * const name = new BemEntityName({ block: 'button' });
+     *
+     * console.log(name); // { block: 'button' }
+     */
+    valueOf() { return this._obj; }
+    /**
+     * Returns object representing the entity name. Need to debug in Node.js.
+     *
+     * In Node.js, `console.log()` calls `util.inspect()` on each argument without a formatting placeholder.
+     * This method will be called to get custom string representation of the object.
+     *
+     * The representation object contains only `block`, `elem` and `mod` fields
+     * without private and deprecated fields (`modName` and `modVal`).
+     *
+     * @returns {Object}
+     * @example
+     * const BemEntityName = require('bem-entity-name');
+     * const name = new BemEntityName({ block: 'button' });
+     *
+     * console.log(name); // { block: 'button' }
+     */
+    inspect() { return this._obj; }
```

Comment:
This is not fair. Better to show that this object is not a primitive, but exactly BemEntityName with some data (_obj).
Can we do it like Error?
```js
> console.log(new Error('zxczxc'))
[Error: zxczxc]
```
```
> console.log(new BemEntityName({ block: 'button' }));
[BemEntityName: {block: 'button'}]
```